script/utils.js

```diff
@@ -12,6 +12,7 @@ const runCommand = (cmd, forcedExitCode = null) => {
   });
 };
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
Added this because `forcedExitCode` is unused on the next line. @bkjohnson - any idea if that param is actually fine to remove?

Reply:
`forcedExitCode` appears to be unused within the `runCommandSync()` function and should be removed https://github.com/department-of-veterans-affairs/vets-website/pull/14072

Reply:
Yeah, saw that PR. Just waiting on it to be merged so I can pull it into this branch.